app/serializers/users/show_serializer.rb

```diff
@@ -0,0 +1,22 @@
+module Users
+  class ShowSerializer < ActiveModel::Serializer
+    attributes :id, :first_name, :last_name, :email, :image_url,
+               :rents_counter, :comments_counter, :unread_notifications_count
+
+    def image_url
+      nil
+    end
+
+    def rents_counter
+      0
```

Comment:
why do you have a counter that always will be 0? 🤔

Reply:
same 👆